### PR TITLE
Limit body size to Github API max length

### PR DESCRIPTION
**Pull request by neongrau**
Otherwise issue creation will fail with an exception of it's own.

### CODE REVIEW

File: lib/ruby_git_issue.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@ def client
   def generate_issue(column_id=nil)
     existing_issue = client.list_issues("#{organization}/#{repo}").select{|issue| issue[:title] == issue_options[:title]}.try(:first)
     if existing_issue.nil?
-      issue_data = client.create_issue("#{organization}/#{repo}", issue_options[:title][0..255], compose_body(exception_data, request), issue_options)
+      issue_data = client.create_issue("#{organization}/#{repo}", issue_options[:title][0..255], compose_body(exception_data, request)[0..65535], issue_options)
```
Comment (arpitvaishnav) [inline lib/ruby_git_issue.rb:35]: Please add the same on the method 
  `def compose_body(e, request)`

The reason is, we are using it for comment as well as create issue

Comment (arpitvaishnav) [reply lib/ruby_git_issue.rb:35]: Also, we need to update version in case of any change on gem

**Comment by arpitvaishnav** [comment]
#1 #2

File: lib/ruby_git_issue.rb (hunk 1 above)
Comment (neongrau) [reply lib/ruby_git_issue.rb:35]: made a fork and fixed it there with new PR

**Pull request closed without merging**